packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -204,7 +205,12 @@ export class PostPopovercontainer {
 
     const isOpen = e.newState === 'open';
     if (isOpen) {
+      const content = this.host.querySelector('.popover-content');
       this.startAutoupdates();
+      if (content && this.animation === `animate-${this.animation}`) {
```

Comment:
`this.animation` can never be equal to `animate-${this.animation}`. 

Please check on [line 99](https://github.com/swisspost/design-system/pull/5586/files#diff-df6410ed94624cc721ffe1939a316d2e4d2de16b0dc709ddacfc2c2fbc56c46dR99) the values `this.animation` can take.